src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -79,7 +80,7 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv)
             }
 
             if (!isset($array[$key]) && !\array_key_exists($key, $array)) {
-                throw new EnvNotFoundException(sprintf('Key "%s" not found in %s (resolved from "%s").', $key, json_encode($array), $next));
+                throw new EnvNotFoundException(sprintf('Key "%s" not found in "%s" (resolved from "%s").', $key, json_encode($array), $next));
```

Comment:
Sorry, not related to the feature, but fixed that because fabbot asked me so.

Reply:
Should be reverted as this is a false positive.

Reply:
Done, thanks.